cls/SourceControl/Git/WebUIDriver.cls

```diff
@@ -5,144 +5,148 @@ ClassMethod HandleRequest(pagePath As %String, InternalName As %String = "", Out
 {
     set context = ##class(SourceControl.Git.PackageManagerContext).ForInternalName(InternalName)
     kill %data
-    #dim %response As %CSP.Response
-    #dim %request As %CSP.Request
-    set pathStart = $Piece(pagePath,"/",2)
+    #dim %response as %CSP.Response
+    #dim %request as %CSP.Request
+    set pathStart = $piece(pagePath,"/",2)
     if pathStart = "api" {
         set handled = 1
         set %data = ##class(%Stream.FileCharacter).%New()
-        if $Extract(pagePath,6,*) = "userinfo" {
+        if $extract(pagePath,6,*) = "userinfo" {
             set responseJSON = ..UserInfo()
-        } elseif $Extract(pagePath,6,*) = "uncommitted" {
+        } elseif $extract(pagePath,6,*) = "uncommitted" {
             set responseJSON = ..Uncommitted()
         } else {
             set %response.Status = ##class(%CSP.REST).#HTTP404NOTFOUND
             set responseJSON = {"error":("invalid URI: " _ pagePath)}
         }
-        if $IsObject($Get(responseJSON)) {
+        if $isobject($get(responseJSON)) {
             do responseJSON.%ToJSON(%data)
         }
-    } elseif $Match(pathStart,"git|dirname|hostname|viewonly") {
+    } elseif $match(pathStart,"git|dirname|hostname|viewonly") {
         if (%request.Method = "GET") {
             set %data = ##class(%Stream.TmpCharacter).%New()
             // Things not handled from Python backend:
             /*
             def do_GET(self):
-                elif self.is_git_request():
-                    self.process_http_backend()
-                else:
-                    SimpleHTTPRequestHandler.do_GET(self)
+            elif self.is_git_request():
```

Comment:
Kind of annoying what happened to the commented-out python equivalent here (especially since spacing matters in that context)... Maybe it's just time to kill these comments.

Reply:
Do we still need this code? If so, we should go back and make an issue for it and embed the code in that. We can remove it from here then